osu.Game/Screens/Edit/Setup/LabelledFileChooser.cs

```diff
@@ -35,7 +32,15 @@ internal class FileChooserLabelledTextBox : LabelledTextBoxWithPopover, ICanAcce
         [Resolved]
         private OsuGameBase game { get; set; } = null!;
 
-        public FileChooserLabelledTextBox(params string[] handledExtensions)
+        private readonly BindableWithCurrent<string> current = new BindableWithCurrent<string>();
+
+        public override Bindable<string> Current
+        {
+            get => current.Current;
+            set => current.Current = value;
+        }
```

Comment:
I am sure there's a reason you added this, but I don't like it.

It doesn't even look like it's hooked up with the underlying component anymore.

Reply:
Probably need to add a comment there, but the idea here is to keep the `TextBox` for displaying placeholder text, but not update it with the filename in `Current`. I get that it's pretty bad, and have mentioned that in OP:

> In addition, the file chooser no longer shows the filename since it's constant now. I wasn't sure how to approach this appropriately but went with the simplest method of overriding `LabelledComponent.Current` and detaching it from the underlying `TextBox` (I've looked into replicating the placeholder text UI but fell into aligning issues instead).

Reply:
I dunno, try and find a better way. maybe a `BindValueChanged` that overrides the value that was set. Or maybe just show the actual filename (`audio.mp3`) along with the text to click to change it.

Reply:
Have decoupled from `LabelledTextBoxWithPopover` and used `PopoverTextBox` implicitly instead, also added xmldoc to explicitly note that the component isn't meant to be coupled with `PopoverTextBox`.